ntropy_sdk/ntropy_sdk.py

```diff
@@ -396,7 +398,10 @@ def retry_ratelimited_request(
                     method,
                     self.base_url + url,
                     json=payload,
-                    headers={"X-API-Key": self.token},
+                    headers={
+                        "X-API-Key": self.token,
+                        "X-SDK-Version": __version__,
```

Comment:
Maybe utilize the `User-Agent` header here including the version there? Then we can easily have a single graph in Grafana to understand API audience.

Reply:
something like this:
```suggestion
                        "User-Agent": f"ntropy-sdk/{__version__}",
```